packages/eslint-config-small-ads/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "eslint-config-small-ads",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Mind writing a short description of that package here?

Reply:
I tried to find best practices on line, there is not much info though. I tried to be short and declarative. But I will listen to any suggestion you may come up.